sandbox

```diff
@@ -214,6 +228,7 @@ sandbox () {
   dc_pty () {
     # note: cannot be replaced by $WINDOWS_PTY_PREFIX dc "$@"
     #       because winpty requires an executable as argument
+    # TODO: can windows docker handle --verbose?
```

Comment:
Are there existing windows build tests?

Reply:
No. Should we introduce?

@michaeldiamant @winder